pom.xml

```diff
@@ -3,7 +3,7 @@
   <modelVersion>4.0.0</modelVersion>
   <groupId>org.opensrp.dashboard</groupId>
   <artifactId>opensrp-dashboard</artifactId>
-  <version>0.0.1-SNAPSHOT</version>
+  <version>0.0.2-SNAPSHOT</version>
```

Comment:
We don't need to update the snapshot version since there are no code updates to the library, only pipeline updates for CI